deploy/operator.yaml

```diff
@@ -29,10 +53,14 @@ spec:
           image: "securityoperators/seccomp-operator:latest"
           imagePullPolicy: Always
           volumeMounts:
-          - mountPath: /var/lib/kubelet/seccomp
-            name: seccomp-root-path
+          - mountPath: /var/lib/kubelet
+            name: kubelet-root-path
+          securityContext:
+            allowPrivilegeEscalation: false
```

Comment:
That’s the default right? If so I guess we can remove it.

Reply:
It actually defaults to true: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#privilege-escalation

Reply:
Whoot.